hoomd/md/HOOMDMath.h

```diff
@@ -0,0 +1,830 @@
+// Copyright (c) 2009-2021 The Regents of the University of Michigan
```

Comment:
You should modify the existing `hoomd/HOOMDMath.h`. Don't create a new identical one in the `md` directory.